napari/_qt/qt_viewer.py

```diff
@@ -136,35 +156,40 @@ def screenshot(self, region=None, size=None, bgcolor=None):
         """
         return self.canvas.render(region, size, bgcolor)
 
-    def _open_images(self):
-        """Adds image files from the menubar."""
+    def _open_files(self):
+        """Adds files from the menubar."""
         filenames, _ = QFileDialog.getOpenFileNames(
             parent=self,
-            caption='Select image(s)...',
+            caption='Select image(s).../volume(npy or npz files)',
             directory=self._last_visited_dir,  # home dir by default
         )
         self._add_files(filenames)
 
     def _add_files(self, filenames):
```

Comment:
Can you revert all these changes, we're not ready yet to support automatically adding a volume through the drop downs. For now we should only support the `add_volume` api but make sure that works as expected

Reply:
done! camera won't work though